tensorbay/client/struct.py

```diff
@@ -85,30 +85,34 @@ class Commit(AttrsMixin, ReprMixin):
         commit_id: The commit id.
         parent_commit_id: The parent commit id.
         message: The commit message.
+        description: The commit description.
         committer: The commit user.
 
     """
 
     _T = TypeVar("_T", bound="Commit")
 
-    _repr_attrs: Tuple[str, ...] = ("parent_commit_id", "message", "committer")
+    _repr_attrs: Tuple[str, ...] = ("parent_commit_id", "message", "description", "committer")
     _repr_maxlevel = 2
 
     commit_id: str = attr(is_dynamic=False, key=camel)
     parent_commit_id: Optional[str] = attr(is_dynamic=False, key=camel)
     message: str = attr(is_dynamic=False)
+    description: Optional[str] = attr(is_dynamic=False)
```

Comment:
looking for all SDK repo, `description` should be `str` and default value is `""`?